polyphemus/lib/etls/ipi/ipi_etls.rb

```diff
@@ -47,9 +47,7 @@ def add_single_cell_linkers!
       end
 
       def single_cell_non_pooled_linker
-        SingleCellLinker.new(
-          record_name_regex: /.*\/(?<record_name>IPI[^\/]*)\/.*$/
-        )
+        SingleCellLinker.new
```

Comment:
This becomes much simpler once I refactored the ipi single cell linker's record name regex to be shared.  In truth, pooled and non pooled linkers are the same, but I kept this separate for now in case other adjustments become obvious.